cd/python/pypi/pypi_publish.py

```diff
@@ -47,14 +48,14 @@ def post_wheel(path):
         print('In DEV account, skipping publish')
         print('Would have run: {}'.format(cmd))
         return 0
-    else:
+    elif any(test_version_mark in version for test_version_mark in ['a', 'b', 'dev']):
```

Comment:
How is this condition related to `See https://github.com/pypa/pypi-support/issues/50 for details`?

Reply:
only rc is allowed for pre-releases.

Reply:
You can consider adding a comment to clarify.